pkg/rbd/rbd_util.go

```diff
@@ -192,13 +167,8 @@ func rbdStatus(pOpts *rbdVolume, userID string, credentials map[string]string) (
 		return false, "", err
 	}
 
-	mon, err := getMon(pOpts, credentials)
-	if err != nil {
-		return false, "", err
-	}
-
-	klog.V(4).Infof("rbd: status %s using mon %s, pool %s", image, mon, pOpts.Pool)
-	args := []string{"status", image, "--pool", pOpts.Pool, "-m", mon, "--id", userID, "--key=" + key}
+	klog.V(4).Infof("rbd: status %s using mon %s, pool %s", image, pOpts.Monitors, pOpts.Pool)
+	args := []string{"status", image, "--pool", pOpts.Pool, "-m", pOpts.Monitors, "--id", userID, "--key=" + key}
```

Comment:
I know you didn't introduce the usage of ```--key``` to all the CLI tools, but is there any concern that the keys are visible to anyone running a "ps"? It would seem like writing the key to a locked-down temp file and using the ```--keyfile``` argument would be a better approach.

Reply:
Yes this is a concern, @phlogistonjohn had pointed this out as well in a separate conversation. This needs to be handled in a subsequent PR, for which a tracking issue does not exist yet (I will create one later today, if one of you hasn't by then).

Reply:
Thanks, you validated my concern -- opened #318